tests/tools/wzdx_translator_test.py

```diff
@@ -65,82 +65,82 @@ def test_parse_xml():
-    actual_icone_data = translator_shared_library.parse_xml(test_input)
+    actual_icone_data = wzdx_translator.parse_xml(test_input)
     assert actual_icone_data == expected_icone_data
 
 
 def test_parse_xml_file_empty_string():
     test_input = ''
     expected_icone_data = None
-    actual_icone_data = translator_shared_library.parse_xml(test_input)
+    actual_icone_data = wzdx_translator.parse_xml(test_input)
     assert expected_icone_data == actual_icone_data
 
 
 def test_parse_xml_file_does_not_exist():
     test_input = 'non-file.txt'
     expected_icone_data = None
-    actual_icone_data = translator_shared_library.parse_xml(test_input)
+    actual_icone_data = wzdx_translator.parse_xml(test_input)
     assert expected_icone_data == actual_icone_data
 
 
-# --------------------------------------------------------------------------------unit test for parse_arguments function--------------------------------------------------------------------------------
+# # --------------------------------------------------------------------------------unit test for parse_arguments function--------------------------------------------------------------------------------
 
 
-def test_parse_arguments():
-    test_input = '-i inputfile.xml -o outputfile.geojson'
-    test_argv = test_input.split(' ')
-    test_input, test_output = translator_shared_library.parse_arguments(
-        test_argv)
-    assert test_output == test_argv[3] and test_input == test_argv[1]
+# def test_parse_arguments():
```

Comment:
Since we've removed the argument parse method altogether let's just remove these tests as well.

Reply:
Removed